### DISCUSSION TITLE
Install scikit-survival with python 3.8

### DISCUSSION THREAD

**Discussion by camferna** [sebp/scikit-survival#423] [Q&A]
Hello,

In order to be compatible with another package I had to create an environment with python 3.8, but I am not able to install scikit-survival in this environment. 

I tried:

1. conda install -c sebp scikit-survival==0.22

 I get

**UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:**

**Specifications:**

  **- scikit-survival -> python[version='>=3.10,<3.11.0a0|>=3.11,<3.12.0a0|>=3.9,<3.10.0a0']**

**Your python: python=3.8**

So it seems that it is not compatible with python 3.8 but it is not true according to: https://pypi.org/project/scikit-survival/

So I tried an older version, 0.14 for exemple and it is no longer available in the channel:

**PackagesNotFoundError: The following packages are not available from current channels:**

**- scikit-survival==0.14**

2.  pip install scikit-survival

I get

**Installing build dependencies ... error
  error: subprocess-exited-with-error**

Which I guess is the same compatibility with python error, but I cant install the version 0.14 either :-[

Is there any way to know all the requirements for the scikit-survival 0.14 so I can get to install the correct version of the other libraries too? which I am guessing is the problem but I actually don't know. It might also be my ggc ? I have read in other 'how to install in Mac' issues that sometimes is due to the Xcode app. 

Thank you in advance!

Camila

**Comment by sebp** [comment]
Are you using macOS with an ARM CPU? If yes, you need at least Python 
version 3.9.

Regarding installation via pip. Try running pip with the -v flag to get 
a more detailed error message which dependency failed to build.
